cmd/add.go

```diff
@@ -140,7 +140,8 @@ func addLocalHookDir(hookName string, fs afero.Fs) {
 }
 
 func getGitHooksDir() string {
-	cmd := exec.Command("sh", "-c", "git rev-parse --git-common-dir")
+	commandArg := strings.Split("git rev-parse --git-common-dir", " ")
+	cmd := exec.Command(commandArg[0], commandArg[1:]...)
```

Comment:
We have static command here, could you please change it in this way:
```go
cmd := exec.Command("git", "rev-parse", "--git-common-dir")
```

Reply:
Fixed